Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -153,9 +154,16 @@ public enum ErrorId {
 Only modules may be declared abstract.
 ".TrimStart(), Remove(true));
 
-    Add(ErrorId.p_general_traits_beta,
+    Add(ErrorId.p_general_traits_datatype,
       @"
-Use of traits as non-reference types is a beta feature. To engage, use /generalTraits:datatype.
+A newtype extending a trait is not generally supported. The option --general-traits=full causes
+Dafny to allow them in the input, but is not recommended.
+".TrimStart(), Remove(true));
+
+    Add(ErrorId.p_general_traits_full,
+      @"
+Use of traits as non-reference types is supported, but is not yet the default. Until it becomes the
+default, use --general--traits=datatype to engage.
```

Comment:
```suggestion
default, use --general--traits=datatype to enable it.
```

(as much as I like the perhaps-accidental allusion to Picard :)